tests/test_convertors.py

```diff
@@ -27,8 +27,8 @@ def to_string(self, value: datetime) -> str:
 
 
 class RegexConvertor(Convertor):
-    def __init__(self, *args):
-        self.regex = args[0]
+    def __init__(self, *args: typing.Any) -> None:
+        self.__class__.regex = args[0]
```

Comment:
The way to suppress mypy error is here: https://github.com/encode/starlette/pull/1455/commits/9d791ff4ea439d383d0051f5b1bd3edd8ae7b6e1#r795218143